### ISSUE TITLE
Update an order

### ISSUE THREAD

**Issue by michaelgold** [orders-squad/orders#6]
**As a** web administrator 
**I need** to be able to update an order  
**So that** I can make a change when an order is incorrectly processed 

**Assumptions:** 
* The wrong information about an order can be corrected
* Display the updated orders

**Acceptance Criteria:**
```
Given update an order
When the order doesn't process correctly
Then the order can be edited and updated

**Comment by michaelgold** [comment]
I suggest rewording this story:

As a website administrator
I need to be able to update an order
So that I can make a manual change if an order is incorrectly processed

**Comment by michaelgold** [comment]
I think the effort level here is a 5

**Comment by 75divisha** [comment]
I think the effort level here is a 5

**Comment by LihengGong** [comment]
Agree with 5.

**Comment by LihengGong** [comment]
`http PUT :5000/orders/1 prod_name=test prod_id=3 cust_id=2 price=3.5 status=test`
```
HTTP/1.0 200 OK
Content-Length: 125
Content-Type: application/json
Date: Tue, 16 Oct 2018 03:08:39 GMT
Server: Werkzeug/0.14.1 Python/2.7.12

{
    "created_on": "Tue, 16 Oct 2018 02:49:47 GMT",
    "cust_id": 2,
    "id": 1,
    "price": 3.5,
    "prod_id": 3,
    "prod_name": "test",
    "status": "test"
}

**Comment by LihengGong** [comment]
This feature has already worked in the latest code.
